src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/timeseries/ComputedTilesToComputedTrends.scala

```diff
@@ -0,0 +1,45 @@
+package com.microsoft.partnercatalyst.fortis.spark.analyzer.timeseries
+
+import com.microsoft.partnercatalyst.fortis.spark.dto.{ComputedTile, ComputedTrend}
+import org.apache.commons.math3.stat.regression.SimpleRegression
+import org.apache.spark.rdd.RDD
+
+object ComputedTilesToComputedTrends {
+
+  def apply(computedTiles: RDD[ComputedTile]): RDD[ComputedTrend] = {
+    val groupedTiles = computedTiles.groupBy(tileGroup)
+    groupedTiles.flatMap[ComputedTrend]((group) => {
+      val tiles = group._2.toSeq
+
+      val regressionResult = linearRegression(tiles)
+
+      val score = regressionResult.getSlope
+      if (score.isNaN) Seq[ComputedTrend]()
+      else {
+        val latestTile = tiles.sortBy(t => -t.periodstartdate).head
+        val filteredTiles = tiles.filter(t => t.period == latestTile.period)
```

Comment:
Is there a more descriptive name than `filteredTiles`? We already know they're filtered from the filter call on the right hand side. E.g. `tilesInLatestPeriod`?